assets/stylesheets/navbar.css

```
@@ -0,0 +1,69 @@
+ul {
+	-webkit-user-select: none;
+    user-select: none;
+    position: fixed;
+    list-style-type: none;
+    padding: 0;
+    background-color: #333;
+    margin: 0;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 60px;
+}
+
+li {
+    float: left;
+}
+
+/* dont worry my css is bad too :) */
+
+li a {
+    display: block;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    color: azure;
+    font-weight: bolder;
+    font-size: 24px;
+    text-align: center;
+    padding: 14px 32px;
+    text-decoration: none;
+    border-radius: 8px;
+}
```

Fix button height (it often either overflows or doesn't cover the whole navbar height)

```suggestion
li:not(#signature) {
    float: left;
    height: 100%;
}

/* dont worry my css is bad too :) */

li a {
    display: flex;
    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
    color: azure;
    font-weight: bolder;
    font-size: 24px;
    text-align: center;
    padding: 14px 32px;
    text-decoration: none;
    border-radius: 8px;
    box-sizing: border-box;

    justify-content: center;
    align-items: center;

    height: 100%;
}
```